packages/bundle-size/README.md

```diff
@@ -24,6 +24,31 @@ export default {
 };
 ```
 
+### Configuration
+
```

Comment:
```suggestion

For custom advanced behavior of `bundle-size`, you can create a `bundle-size.config.js` in the root of your project directory (next to `package.json`).

```

Is there sense?

Reply:
I though the file tree was obvious enough, but adding description doens't hurt